examples/change-element-style/README.md

```diff
@@ -0,0 +1,27 @@
+# Change Element's Style
+
+**Test Code**: [change-elements-style.js](change-elements-style.js)
+
+This example shows how to modify an element's style from test code.
+
+The [change-elements-style.js](change-elements-style.js) file contains two tests. These tests use [client functions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html) to run code in the browser.
+
+During the `Hide an element` test, the `hidePopulateButton` client function sets the `#populate` button's  [style.display](https://developer.mozilla.org/en-US/docs/Web/CSS/display) CSS property to `none`. Before the function is called, the [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) assertion verifies that the button is visible. After `hidePopulateButton` is executed, the [t.expect.notOk](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/notok.html) assertion ensures that the button is no longer displayed.
```

Comment:
`the [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) assertion **method** verifies`

here and below